### PR TITLE
feat:Adding funct

### PR DESCRIPTION
**Pull request by ashishprodev**
Create MR

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-  Button, ScrollView
+  StyleSheet, View,
 } from 'react-native'
 import Header from './Component/Header'
 import TaskManager from './Component/TaskManager'
 import Todolist from './Component/Todolist'
 import React from 'react'
 
-const App = () => {
+const App = ({}) => {
```
Comment (hangingPandaReact) [inline App.js:9]: why are you using ({}) in here

File: App.js [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
   return (
     <View  >
       <Header />
+      {/* <InputHandler /> */}
```
Comment (hangingPandaReact) [inline App.js:13]: remove unuse code

File: Component/Header.js [hunk 3]
```diff
@@ -1,19 +1,33 @@
...
       <SafeAreaView >
       <View style={styles.mainContainer}>
           <Image source={require('../avator.png')}
               style={styles.img}
           />
         <TouchableOpacity style={styles.Btn}>
-            <Text style={{fontSize:25,color:'white'}}>+</Text>
-        </TouchableOpacity>
+                    <Text
+                        style={{ fontSize: 25, color: 'white' }}
```
Comment (hangingPandaReact) [inline Component/Header.js:28]: move style to styles variable

File: App.js (hunk 2 above)
Comment (ashishprodev) [reply App.js:13]: removed components

**Pull request merged**